frontend/src/component/admin/groups/GroupForm/GroupFormUsersSelect/GroupFormUsersSelect.tsx

```diff
@@ -101,7 +102,7 @@ export const GroupFormUsersSelect: VFC<IGroupFormUsersSelectProps> = ({
 
     return (
         <StyledGroupFormUsersSelect>
-            <Autocomplete
+            <Combo
```

Comment:
Nice drop-in replacement should make us happy coders and provide a DRYer approach 🙂